skyportal/plot.py

```diff
@@ -516,33 +516,87 @@ def make_clear_and_add_buttons(model_dict):
     )
 
 
-def make_add_filter_group_form(split, model_dict, panel_name):
+def make_add_filter_group_form(split, model_dict, panel_name, user):
     labels = [label for label, sdf in split]
-    checkboxes = CheckboxGroup(labels=labels, active=[], width=100)
+    label_final = set()
+    for i in labels:
+        label_final.add(i.split('/')[1])
+        label_final.add(i.split('/')[2])
+    checkboxes = CheckboxGroup(labels=list(label_final), active=[], width=100)
```

Comment:
Cool. I understand why my labels don't make sense now. Maybe we should split these into two checkbox groups, one for filters and one for origins.

Reply:
Yes, I think it's a good idea. It will be much clearer this way

Reply:
@thomasculino If you give me write access to your skyportal fork, I can add this.

Reply:
I added it. This is how it looks now:
![Peek 2022-04-12 09-49](https://user-images.githubusercontent.com/60301767/162909603-d847925f-9862-482a-a9b1-68b605ed7ca7.gif)